app/src/main/java/com/metro/gwuexplorer/MapsActivity.kt

```diff
@@ -0,0 +1,43 @@
+package com.metro.gwuexplorer
+
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.android.gms.maps.model.MarkerOptions
+
+class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
```

Comment:
You won't actually need to do a `MapsActivity` for the project (unless you want to) 😄